Gemfile.lock

```diff
@@ -580,9 +582,7 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   unobtrusive_flash (>= 3)
-  web-console (~> 2.0)
-  webmock
-  whenever
```

Comment:
I did it manually. I though that it was strange to remove them and I saw that travis failed because of that. I clearly misunderstand what @ChrisBr said. I'll fix it. Thank you. :wink: